public/main.js

```diff
@@ -554,3 +554,8 @@ function changeAngleCurrentToOriginalCamera(angle, centerPosition){
     X_AXIS_camera = new THREE.Vector3( 1, 0, 0 );
     Z_AXIS_camera = new THREE.Vector3( 0, 0, -1 );
 }
+
+function showHelp() {
+    var help = document.getElementById('help-message');
```

Comment:
Every client has a variable 'id' that represents the id of the client. The window in the centre has an id of 1. So in order to only add the help control instructions in the central window, you can add a conditional that checks if the id is 1. You can find the list of variables at the beginning of the js script in index.html.

Reply:
Ah thank you!